packages/eslint-config-dx-team/index.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: ['plugin:@typescript-eslint/eslint-recommended', 'prettier'],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
```

Comment:
If we will not separate such settings to an additional packages, eslint will miss problems in simple projects.
Let's separate for several configs.

Reply:
We use typescript in all of our projects, so I think this specific parser makes most sense. We can remove the `jsx: true` because it is react-specific though. 

Durning integration phase we can consider if splitting this preset makes more sense, WDYT?